src/init-project/index.ts

```diff
@@ -597,6 +598,37 @@ function addLanguagesToConfigurationFiles(host: Tree, options: InitProjectOption
   };
 }
 
+function addApiToConfigurationFiles(host: Tree, options: InitProjectOptions): Rule {
```

Comment:
I think it's useful since we need to re-implement all configuration files using a base configuration file

Reply:
Yes, I meant `useless`, sorry. 

We need to implement configuration files that won't duplicate themselves. It can be achieved if we create `configuration.base.ts` with code:

```ts
import { merge } from 'lodash';

export function initConfiguration(configuration: ...): ... {
  return merge({ ... }, configuration);
```

 and will extend it in other files:

configuration.ts:

```ts
import { initConfiguration } from './configuration.base';

export const configuration = initConfiguration({
  ....
});
```

Reply:
Ok. I removed this part